ci/taos/config/config-plugins-audit.sh

```diff
@@ -74,3 +74,14 @@ echo "[MODULE] plugins-staging: Plugin group that does not have evaluation and a
 # $module_name
 # echo "[DEBUG] $module_name is done."
 
+module_name="pr-nnstreamer-apptest"
+echo "[DEBUG] TAOS/$module_name: Check nnstreamer sample app"
+echo "[DEBUG] $module_name is started."
+echo "[DEBUG] Current path: $(pwd)."
+source ${REFERENCE_REPOSITORY}/ci/taos/plugins-staging/$module_name.sh
+# Note that do not append the below "$module_name" because build step is implemented as a built-in module partially
+
+
+###### Add plugins to run ################################################################################################
+# Write plugins to test ex) "pr-nnstreamer-apptest pr-audit-resource" 
+export audit_plugins="pr-nnstreamer-apptest"
```

Comment:
I have two questions.
First, Do we have to use "export" mandatorily?
Second, Can we manage the variable audit_plugins stably  and consistently when we have to enable lots of modules?

Reply:
@ohsewon Please try to variable array instead of `export audit_plugins ...` to minimize user's mistake. 


For example, 
```bash

audit_plugins[1]="pr-audit-build-tizen"
echo "[DEBUG] The default BUILD_MODE of $audit_plugins[1] is declared with 99 (SKIP MODE) by default in plugins-base folder."
echo "[DEBUG] $audit_plugins[1] is started."
echo "[DEBUG] TAOS/$audit_plugins[1]: Check if Tizen rpm package is successfully generated."
echo "[DEBUG] Current path: $(pwd)."
source ${REFERENCE_REPOSITORY}/ci/taos/plugins-base/$audit_plugins[1].sh
# Note that do not append the below "$audit_plugins[1]" because build step is implemented as a built-in module partially

     . . .  OMISSION  . . . 

audit_plugins[4]="pr-nnstreamer-apptest"
echo "[DEBUG] TAOS/$module_name: Check nnstreamer sample app"
echo "[DEBUG] $audit_plugins[4] is started."
echo "[DEBUG] Current path: $(pwd)."
source ${REFERENCE_REPOSITORY}/ci/taos/plugins-staging/$audit_plugins[4].sh
# Note that do not append the below "$audit_plugins[4]" because build step is implemented as a built-in module partially
```

Reply:
* Reference: https://www.linuxjournal.com/content/bash-arrays